types/is-object/is-object-tests.ts

```diff
@@ -0,0 +1,8 @@
+import * as isObject from 'is-object';
```

Comment:
This should not be imported with ES6 syntax. See https://stackoverflow.com/questions/39415661/what-does-resolves-to-a-non-module-entity-and-cannot-be-imported-using-this

The correct form would be `import isObject = require('is-object');`